src/Item/Item.model.ts

```diff
@@ -181,6 +187,26 @@ export const ItemQueries = Object.freeze({
         }
         WHERE items.collection_id = ANY(${collectionIds})
           AND ${status ? SQL`item_curations.status = ${status}` : SQL`1 = 1`}
+        ORDER BY items.created_at ASC
+        LIMIT ${limit}
+        OFFSET ${offset}
+    `,
+  selectByStandardCollectionIds: (
+    collectionIds: string[],
+    status?: CurationStatus,
+    limit: number = DEFAULT_LIMIT,
+    offset: number = 0
+  ) =>
+    SQL`
+      SELECT items.*, count(*) OVER() AS total_count
+        FROM ${raw(Item.tableName)} items
+        WHERE items.collection_id = ANY(${collectionIds})
+          AND ${
+            status && status === CurationStatus.PENDING
+              ? SQL`items.blockchain_item_id IS NULL`
+              : SQL`1 = 1`
+          }
```

Comment:
The `blockchain_item_id` is independent from the curation status as it's set once the collection is published.
Standard items don't have curation statuses, their collection can have (once it changes and is up to re-review) a curation so I wouldn't mix that concept in the query unless we wanted to get the items of a collection with a pending status.

Reply:
Isn't this that case? The `status` is for the items that are getting asked.
If we're asking for `pending` items, shouldn't we return the ones with `blockchain_item_id` NOT null? I might be missing something here..